lib/run-contentful-export.js

```diff
@@ -22,6 +23,15 @@ export default function runContentfulExport (usageParams) {
     spaceId: clients.source.spaceId
   })
     .then((response) => {
+      if (opts.downloadAssets) {
+        log.info('Downloading ' + response.assets.length + ' assets')
+        response.assets.forEach((asset) => {
+          Object.keys(asset.fields.file).forEach((key) => {
+            downloadAsset(asset.fields.file[key].url, opts.exportDir)
+          })
+        })
+        log.info('All ' + response.assets.length + ' downloads finished')
```

Comment:
even if it is super rare, that we won't get all the assets, could happen though when you don't have enough space on disk or bad internet etc..., it would be nice if you handle errors, count the successfully downloaded assets instead of outputing assets.length, and log failing ones. This way the user can keep track of the missing assets